packages/core/src/routes/+page.svelte

```diff
@@ -23,3 +24,6 @@ import Badge from '$lib/badge.svelte';
   variant="blue"
   label="Info"
 />
+
+<!-- Breadcrumbs -->
+<Breadcrumbs crumbs={['Chocolate Chip', 'Oatmeal Raisin']} />
```

Comment:
Maybe we should start splitting this into individual routes?

Reply:
I could go either way, let's start here and split once it becomes unruly?